airbyte-integrations/connectors/source-hubspot/source_hubspot/streams.py

```diff
@@ -193,22 +185,23 @@ def _parse_and_handle_errors(response) -> Union[MutableMapping[str, Any], List[M
             message = response.json().get("message")
 
         if response.status_code == HTTPStatus.BAD_REQUEST:
-            message = f"Request to {response.url} didn't succeed. Please verify your credentials and try again.\nError message from Hubspot API: {message}"
-            raise HubspotBadRequest(internal_message=message, failure_type=FailureType.config_error, response=response)
+            logger.warning(
+                f"Request to {response.url} didn't succeed. Please verify your credentials and try again.\nError message from Hubspot API: {message}"
+            )
         elif response.status_code == HTTPStatus.FORBIDDEN:
-            message = f"The authenticated user does not have permissions to access the URL {response.url}. Verify your permissions to access this endpoint."
-            raise HubspotAccessDenied(internal_message=message, failure_type=FailureType.config_error, response=response)
+            logger.warning(
+                f"The authenticated user does not have permissions to access the URL {response.url}. Verify your permissions to access this endpoint."
+            )
         elif response.status_code in (HTTPStatus.UNAUTHORIZED, CLOUDFLARE_ORIGIN_DNS_ERROR):
             message = (
                 "The user cannot be authorized with provided credentials. Please verify that your credentails are valid and try again."
             )
-            raise HubspotInvalidAuth(internal_message=message, failure_type=FailureType.config_error, response=response)
```

Comment:
Why you delete failure_type config error?

Reply:
removed it by mistake, added back